python/build.py

```diff
@@ -39,13 +39,18 @@ def build(project_name):
         f'{get_python_executable()} ../misc/make_changelog.py origin/master ../ True'
     )
 
+    os.environ['PROJECT_NAME'] = project_name
```

Comment:
Nit: it'd be nice if we can add a comment saying this env is used by `setup.py` below. It's easy to forget updating this env when we change setup.py below.